rocketpy/rocket/rocket.py

```diff
@@ -1493,16 +1502,36 @@ def add_parachute(
             The values are used to add noise to the pressure signal which is
             passed to the trigger function. Default value is (0, 0, 0). Units
             are in pascal.
+        parachute_radius : float, optional
+            Radius of the inflated parachute. Default value is 1.5.
+            Units are in meters.
+        parachute_height : float, optional
+            Height of the inflated parachute. Default value is the radius parachute.
+            Units are in meters.
+        porosity : float, optional
+            Porosity of the parachute material, which is a measure of how much air can
+            pass through the parachute material.
+            Default value is 0.0432 (for consistency with previous versions).
```

Comment:
Could you define more precisely what porosity is? Something like: "The ratio of the area of the porous structure to the total area of the canopy" (I don't know if this is the correct definition in our case, just an example)